.github/workflows/smoke-test.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Smoke Tests ${{ github.event.inputs.environment }}
-        run: echo "Nothing to run yet 🚀"
+        run: bundle exec rspec --tag smoke
```

Comment:
Do we need to set the `RAILS_ENV` here to make sure `Settings.base_url` is returned correctly?

Reply:
have added in https://github.com/DFE-Digital/register-trainee-teachers/pull/372/commits/e8b8f6cc4127b370e70bd3f0385b7313a0d336c2